ogc/styles/main.go

```diff
@@ -103,20 +114,25 @@ func (s *Styles) Styles() http.HandlerFunc {
 
 func (s *Styles) Style() http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
-		styleID := chi.URLParam(r, "style")
+		style := chi.URLParam(r, "style")
+		styleID := strings.Split(style, "__")[0]
+		// backwards compatibility
+		if style == styleID {
```

Comment:
Misschien kan je dat nog iets duidelijker maken (code of comment) hoe dit voor backwards compat zorgt. Idee is dat de huidige style URLs blijven werken + dat we een style per projectie hebben en die laatste ook adverteren in de API